src/Assignments/Assignment-3/Ques2/PreferenceContext.jsx

```diff
@@ -0,0 +1,15 @@
+import { createContext, useState } from "react"
+
+export const PreferenceContext = createContext();
+export const PreferenceProvider = ({children})=>{
+    const [theme, setTheme] = useState("light");
+    const switchTheme = () => {
+        if (theme === "light") setTheme("dark");
+        else setTheme("light");
+      };
```

Comment:
add light, dark words in constant file. constant file variables should be in capital letters 
DARK:"dark"